src/arviz_plots/plot_matrix.py

```diff
@@ -158,6 +160,48 @@ def _generate_viz_dt(self, **figure_kwargs):
             )
         )
 
+    @property
+    def orientation(self):
+        """Orientation of targets we want to fetch through :meth:`~PlotMatrix.get_target`.
+
+        Orientation is always taken into account when using
+        :meth:`~PlotMatrix.get_target` to fetch target for plotting.
+        """
+        return self._orientation
+
+    @orientation.setter
+    def orientation(self, value):
+        self._orientation = value
+
+    @property
+    def fixed_var_name(self):
+        """Fixed variable's name of targets we fetch through :meth:`~PlotMatrix.get_target`.
+
+        fixed_var_name is taken into account when using :meth:`~PlotMatrix.get_target`
+        to fetch target for plotting and :attr:`~PlotMatrix.orientation` is
+        set to either `row` or `col`.
+
+        It tells the fixed variable name in a `row` or `col`
+        """
+        return self._fixed_var_name
+
+    @fixed_var_name.setter
+    def fixed_var_name(self, value):
+        self._fixed_var_name = value
+
+    @property
+    def fixed_selection(self):
+        """Fixed dictionary for subsetting of targets."""
+        return self._fixed_selection
+
+    @fixed_selection.setter
+    def fixed_selection(self, value):
+        self._fixed_selection = value
+
+    @fixed_var_name.setter
+    def fixed_var_name(self, value):
+        self._fixed_var_name = value
```

Comment:
I think we could have them as "simple" attributes set to `None` in `__init__`. Moreover, I don't think we expect/want users to interact with them directly but more so as an internal thing to reduce duplication between `map` and `map_row_col`